drivers/snmp-ups.c

```diff
@@ -392,6 +392,19 @@ void upsdrv_initups(void)
 		dstate_addcmd("shutdown.return");
 		dstate_addcmd("shutdown.stayoff");
 	}
+
+	/* Publish sysContact and sysLocation for all subdrivers */
+	/* sysContact.0 */
+	if (nut_snmp_get_str(".1.3.6.1.2.1.1.4.0", model, sizeof(model), NULL) == TRUE)
```

Comment:
Should the OID strings be explicit here, or defined as macros (perhaps in common snmp-ups.h)? Or even (potentially) individualized in mappings if e.g. a particular verndor supports some extended attribute for this (or plain different, ignoring the standards) and NUT might honor it?

Reply:
Finally addressed a few years later ;)